File: src/main/java/be/superjoran/mint/dao/StatementDao.java
package be.superjoran.mint.dao;

import be.superjoran.mint.domain.Person;
import be.superjoran.mint.domain.Statement;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface StatementDao extends JpaRepository<Statement, String> {

    List<Statement> findAllByOriginatingAccount_Owner(Person owner);
}

File: src/main/java/be/superjoran/mint/dao/StatementDao.kt
package be.superjoran.mint.dao

import be.superjoran.mint.domain.Person
import be.superjoran.mint.domain.Statement
import org.springframework.data.jpa.repository.JpaRepository

interface StatementDao : JpaRepository<Statement, String> {

    fun findAllByOriginatingAccount_Owner(owner: Person): List<Statement>
}
